src/main/java/org/folio/consortia/domain/entity/Tenant.java

```diff
@@ -0,0 +1,35 @@
+package org.folio.consortia.domain.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.hibernate.Hibernate;
+
+import java.util.Objects;
+
+@Getter
+@Setter
+@ToString
+@RequiredArgsConstructor
+@Entity
+public class Tenant {
+  @Id
+  private String id;
+  private String name;
+
+  @Override
+  public boolean equals(Object o) {
```

Comment:
I know that is some cons using lombock to implement equals and hashcode in JPA mapping, so because of this you decided to not use lombock here?

Reply:
![image](https://user-images.githubusercontent.com/113523904/224475478-08908498-77f6-4ec3-81d3-8ce6b5a23ad2.png)
it is giving warn message